app/error_messages.py

```diff
@@ -5,6 +5,7 @@
     "The referenced answer cannot be used to set the minimum of answer"
 )
 ANSWER_TYPE_FOR_OPTION_LABEL_FROM_VALUE_INVALID = "The answer type for option label from value is not of type ['Radio','Checkbox','Dropdown']"
+ANSWER_TYPE_FOR_TRANSFORM_TYPE_INVALID = "The answer type for 'format_percentage' or 'format_unit' transform is not of type 'Percentage','Unit'"
```

Comment:
This should be specific. We want to know if it was one or the other etc. I would make this a str like:

```py
ANSWER_TYPE_FOR_TRANSFORM_TYPE_INVALID = "The answer type for '{transform}' transform is not of type '{answer_type}'"
```

Then you can just use `.format()` when calling this to pass the relevant info.

Reply:
Ok, added and using `.format()`.